### ISSUE TITLE
Connection Closed: Clock Gettime

### ISSUE THREAD

**Issue by RAbraham** [luna/luna#154]
Steps to reproduce:
* I downloaded Luna 1.2 and clicked on the Basics Tutorial

Error 
```
Connection closed.

======= logs: =======

=== luna-empire-stdout---supervisor-Q4MSDx.log ===
DEBUG Loading Luna configuration
DEBUG Loading Luna configuration
DEBUG Connecting to bus...
dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
  Referenced from: /Applications/LunaStudio.app/Contents/Resources/1.2/bin/private/../../lib/libzmq.5.dylib
  Expected in: flat namespace

dyld: Symbol not found: _clock_gettime
  Referenced from: /Applications/LunaStudio.app/Contents/Resources/1.2/bin/private/../../lib/libzmq.5.dylib
  Expected in: flat namespace

DEBUG Loading Luna configuration
DEBUG Loading Luna configuration
DEBUG Connecting to bus...
dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
  Referenced from: /Applications/LunaStudio.app/Contents/Resources/1.2/bin/private/../../lib/libzmq.5.dylib
  Expected in: flat namespace

dyld: Symbol not found: _clock_gettime
  Referenced from: /Applications/LunaStudio.app/Contents/Resources/1.2/bin/private/../../lib/libzmq.5.dylib
  Expected in: flat namespace

DEBUG Loading Luna configuration
DEBUG Loading Luna configuration
DEBUG Connecting to bus...
dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
  Referenced from: /Applications/LunaStudio.app/Contents/Resources/1.2/bin/private/../../lib/libzmq.5.dylib
  Expected in: flat namespace

dyld: Symbol not found: _clock_gettime
  Referenced from: /Applications/LunaStudio.app/Contents/Resources/1.2/bin/private/../../lib/libzmq.5.dylib
  Expected in: flat namespace

DEBUG Loading Luna configuration
DEBUG Loading Luna configuration
DEBUG Connecting to bus...
dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
  Referenced from: /Applications/LunaStudio.app/Contents/Resources/1.2/bin/private/../../lib/libzmq.5.dylib
  Expected in: flat namespace

dyld: Symbol not found: _clock_gettime
  Referenced from: /Applications/LunaStudio.app/Contents/Resources/1.2/bin/private/../../lib/libzmq.5.dylib
  Expected in: flat namespace
```
Environment and other possible hints
* Mac OS. El Capitan 10.11.6
* I have Atom installed for my other dev work
* I had installed Luna Studio previously and then I tried to update the Atom plugins that it was asking me to and things failed. So I uninstalled and installed again and I got the above error.

**Comment by mikusp** [comment]
Thank you for the report. Did Luna Studio work before you tried to update the Atom plugins?

**Comment by Spaceghost** [comment]
I found [this][] earlier. You may need to upgrade to Sierra.

[this]: https://stackoverflow.com/questions/45833569/dyld-lazy-symbol-binding-failed-symbol-not-found-clock-gettime-in-mongodb/45933407#45933407

**Comment by RAbraham** [comment]
@mikusp I'm sorry I don't remember. I think I clicked into the basic tutorial and before I could go forward, I think I saw the notification to update and I did and it went wrong. Note this is before the above ticket. I uninstalled when things went wrong and I reinstalled and I got the above error. 
@Spaceghost that's a good find. I'll try from my personal laptop which is on Sierra and report back but that will be in a few weeks or so :)
